modules/publish/templates/_leftmenu.inc.php

```diff
@@ -1,6 +1,6 @@
 <?php include_component('main/menulinks', array('links' => $links, 'target_type' => 'wiki', 'target_id' => $links_target_id, 'title' => __('Wiki menu'))); ?>
 <?php if ($article instanceof \thebuggenie\modules\publish\entities\Article && $article->getID()): ?>
     <?php include_component('publish/whatlinkshere', compact('article')); ?>
-    <?php include_component('publish/tools', compact('special', 'article')); ?>
+    <?php include_component('publish/tools', compact(isset($special)?'special':array(), 'article')); ?>
```

Comment:
See previous comments about initializing the methods in the associated component method